package.json

```diff
@@ -31,7 +31,9 @@
     "codecov": "^3.1.0",
     "eslint": "^5.4.0",
     "eslint-config-amk": "^0.1.0",
+    "knex": "^0.15.2",
```

Comment:
I think, this should be either `peerDependency` too, or only normal `dependency`.

Reply:
yup, peer dependency would be better. thanks